src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
 // export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const partner = sex === 'm' ? `wife` : `husband`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      { age && (
```

Comment:
If age is equal to 0, 0 will be rendered instead of the p tag. That's not the behavior we want.
Convert the age to boolean (`!!age`) or check if `age !== undefined`